notebooks/outetts-text-to-speech/outetts-text-to-speech.ipynb

```diff
@@ -22,6 +22,11 @@
     "- [Run model inference](#Run-model-inference)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    r = requests.get(
</pre><p>please add check that "skip_kernel_extension.py" exists to allow rerun notebook without internet connection</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Done.